### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by krHERO**
## Description
as agreed at the edirom community meeting 9.10.2024, we do not want to have tasks but (optional) bullet points.

## Types of changes

- Improvement

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,18 +9,20 @@ Refs #
...
+## Overview
+<!--- Go over all the following points, and DELETE options that are not relevant. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have performed a self-review of my code
-- [ ] I have read the [CONTRIBUTING](https://github.com/Edirom/Edirom-Online/blob/develop/CONTRIBUTING.md) document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- My change requires a change to the documentation.
```
Comment (peterstadler) [inline .github/pull_request_template.md:23]: Regarding documentation: It's still unclear to me where that lives and where to update. So, a link would be helpful for new contributors like me.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -9,18 +9,20 @@ Refs #
...
+- I have updated the documentation accordingly.
+- I have performed a self-review of my code
+- I have read the [CONTRIBUTING](https://github.com/Edirom/Edirom-Online/blob/develop/CONTRIBUTING.md) document.
+- I have added tests to cover my changes.
```
Comment (peterstadler) [inline .github/pull_request_template.md:27]: Regarding tests: For what I know there is no test suite in place for the Edirom Online. So, I do not know how I am supposed to add tests?!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -9,18 +9,20 @@ Refs #
...
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have performed a self-review of my code
-- [ ] I have read the [CONTRIBUTING](https://github.com/Edirom/Edirom-Online/blob/develop/CONTRIBUTING.md) document.
-- [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
+- My change requires a change to the documentation.
+- I have updated the documentation accordingly.
+- I have performed a self-review of my code
```
Comment (peterstadler) [inline .github/pull_request_template.md:25]: What exactly does that mean?

**Review by peterstadler — CHANGES_REQUESTED** [review]
Several points are still unclear to and would benefit from further explanation.

File: .github/pull_request_template.md (hunk 3 above)
Comment (krHERO) [reply .github/pull_request_template.md:25]: - *inline* documentation
- review --> add link styleguides

File: .github/pull_request_template.md (hunk 1 above)
Comment (krHERO) [reply .github/pull_request_template.md:23]: - delete here, eventually add at the ssubsection self review (example code documentation)

File: .github/pull_request_template.md (hunk 2 above)
Comment (krHERO) [reply .github/pull_request_template.md:27]: add link https://github.com/Edirom/Edirom-Online/tree/develop/testing

**Comment by krHERO** [comment]
> Several points are still unclear to and would benefit from further explanation.
thanks for the points you pointed out in your first review. 
could you please have a look again, i changed the stuff, we discussed recently. thanks

**Review by peterstadler — APPROVED** [review]
looks good to me, many thanks!

**Pull request merged**